eHealthAfrica/direct-delivery
# react-native setup android buildTools 25

#### What does this PR do?
Setup v3 branch on `react-native` with the following: (Branch `v3` will serve as dev branch until we are ready to phase out the production app)

- React Redux
- Redux Thunk
- React Native Router Flux
- React Native PouchDB
- Setup Travis builds: 
(APK deployment to an s3 directory is pending. Access required from DevOps)

#### Description of Task to be completed?

#### How should this be manually tested?
Clone repo
Run `npm i -g react-native-cli` then `npm i` then `react-native run-android` 
Make sure your android env is setup with a simulator or connect an android device to your computer

#### Any background context you want to provide?

#### What are the relevant JIRA stories?
[#HDSLD-349](https://jira.ehealthafrica.org/browse/HDSLD-349)
[#HDSLD-350](https://jira.ehealthafrica.org/browse/HDSLD-350)
